homepage/views.py

```diff
@@ -47,7 +48,11 @@ def courses(request):
 
 def reviews(request):
     reviews = Review.main_feed()
-    return render(request, 'homepage/reviews/reviews.html', {'reviews': reviews})
+    if (request.user.is_anonymous):
+        liked_reviews = []
+    else:
+        liked_reviews = UserLikes.get_liked_reviews_by_user(request.user)
```

Comment:
because I saw you didn't like the same comment @oriAdler added
```suggestion
        liked_reviews = [] if request.user.is_anonymous else UserLikes.get_liked_reviews_by_user(request.user)
```

I added one more option:
```suggestion
        liked_reviews = []
        if not request.user.is_anonymous:
             liked_reviews = UserLikes.get_liked_reviews_by_user(request.user)
```
and if you don't like it either so np